### PR TITLE
CDS-78285 kustomize no custom remote manifest support

### PR DESCRIPTION
**Pull request by michaelcretzman**
https://harness.atlassian.net/browse/CDS-78285

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* https://harness.atlassian.net/browse/CDS-78285

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/x-platform-cd-features/services/cd-services-config-files.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ You can use files added to the **Config Files** section in your Harness services
 
 ## Supported platforms
 
+You can store files in the following locations.
+
 ### Harness file store
 
-All platform [integrations](/docs/continuous-delivery/cd-integrations) (Kubernetes, etc) support config files stored in the Harness File Store.
+All platform [integrations](/docs/continuous-delivery/cd-integrations) (Kubernetes, etc.) support config files stored in the [Harness file store]().
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/services/cd-services-config-files.md:15]:
```suggestion
All platform [integrations](/docs/continuous-delivery/cd-integrations) (Kubernetes, etc.) support config files stored in the [Harness file store](/docs/continuous-delivery/x-platform-cd-features/services/add-inline-manifests-using-file-store).
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-quickstart.md [hunk 2]
```diff
@@ -10,46 +10,38 @@ helpdocs_is_published: true
 
 Harness supports [Kustomize](https://kustomize.io/) kustomizations in your Kubernetes deployments. You can use overlays, multibase, plugins, sealed secrets, etc, just as you would in any native kustomization.
 
-This Kustomize tutorial will deploy multiple variants of a simple public Hello World server using a [rolling update strategy](../kubernetes/kubernetes-executions/create-a-kubernetes-rolling-deployment) in Harness.
+This Kustomize overview will describe how to deploy multiple variants of a simple public Hello World server using a [rolling update strategy](../kubernetes/kubernetes-executions/create-a-kubernetes-rolling-deployment) in Harness.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-quickstart.md:13]:
```suggestion
This Kustomize overview describes how to deploy multiple variants of a simple public Hello World server using a [rolling update strategy](../kubernetes/kubernetes-executions/create-a-kubernetes-rolling-deployment) in Harness.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-quickstart.md [hunk 3]
```diff
@@ -10,46 +10,38 @@ helpdocs_is_published: true
...
-## Create the Deploy stage
+## Create the deploy stage for Kustomize
 
-Pipelines are collections of stages. For this quickstart, we'll create a new Pipeline and add a single stage.
+Pipelines are collections of stages. For this topic, we'll create a new pipeline and add a single stage.
 
 :::note
 
-**Create a Project for your new CD Pipeline:** if you don't already have a Harness Project, create a Project for your new CD Pipeline. Ensure that you add the **Continuous Delivery** module to the Project. See [Create Organizations and Projects](/docs/platform/organizations-and-projects/create-an-organization).
+**Create a Project for your new CD Pipeline:** if you don't already have a Harness project, create a project for your new CD pipeline. Ensure that you add the **Continuous Delivery** module to the project. See [Create Organizations and Projects](/docs/platform/organizations-and-projects/create-an-organization).
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-quickstart.md:44]:
```suggestion
**Create a Project for your new CD Pipeline:** if you don't already have a Harness project, create a project for your new CD pipeline. Ensure that you add the **Continuous Delivery** module to the project. For more information, go to [Create Organizations and Projects](/docs/platform/organizations-and-projects/create-an-organization).
```

**Review by brian-f-harness — DISMISSED** [review]
LGTM. Left a couple of minor suggestions.

**Review by thisrohangupta — COMMENTED** [review]
this looks good

**Pull request merged**